controllers/rest/middleware_initialize.go

```diff
@@ -46,9 +46,27 @@ func CreateSchema(ctx context.Context, e *echo.Echo, s *openapi3.Swagger) map[st
 //creates a new schema interface instance
 func newSchema(ref *openapi3.Schema, logger echo.Logger) (ds.Builder, map[string]string, []string) {
 	pks, _ := json.Marshal(ref.Extensions["x-identifier"])
+	dfs, _ := json.Marshal(ref.Extensions["x-remove"])
```

Comment:
please use constants to reference the extension names so that if we decide to rename in the future it would be an update in one place